internal/provisioner/utility/argo_utility.go

```diff
@@ -120,7 +120,7 @@ func ProvisionUtilityArgocd(utilityName, tempDir, clusterID string, allowCIDRRan
 
 	appName := utilityName + "-sre-" + awsClient.GetCloudEnvironmentName() + "-" + clusterID
 
-	timeout := time.Second * 600
+	timeout := time.Second * 420
```

Comment:
nit: adding a constant for this timeout will make future changes easier (only have to change the one value)